draft-tuexen-tsvwg-rfc4960-errata.xml

```diff
@@ -733,6 +733,79 @@ partial_bytes_acked and cwnd in the congestion avoidance phase.</t>
 </section>
 </section>
 
+<section title='HEARTBEAT ACK and the association error counter'>
+<section title='Description of the Problem'>
+<t>Section 8.1 and Section 8.3 of <xref target="RFC4960"/>
+prescribes that the receiver of a HEARTBEAT ACK must reset the
+association overall error counter. In some circumstances, e.g. when
+a router discards DATA chunks but not HEARTBEAT chunks due to the
+larger size of the DATA chunk, it is better to not clear the
+association error counter by the HEARTBEAT ACK and reset it by the
+SACK only.</t>
+</section>
+<section title='Text Changes to the Document'>
+<figure>
+<artwork>
+---------
+Old text: (Section 8.1)
+---------
+
+The counter shall be reset each time a DATA chunk sent to that peer
+endpoint is acknowledged (by the reception of a SACK) or a HEARTBEAT
+ACK is received from the peer endpoint.
+
+---------
+New text: (Section 8.1)
+---------
+
+The counter shall be reset each time a DATA chunk sent to that peer
+endpoint is acknowledged (by the reception of a SACK). When a
+HEARTBEAT ACK is received from the peer endpoint, the counter should
+also be reset. The receiver of the HEARTBEAT ACK may choose not to
+clear the counter while transmitting data on the association. Such
+choice may be motivated by different sizes of the DATA and HEARTBEAT
+chunks. 
```

Comment:
Do you really want to make the decision based on the size? Or would it make sense to just not clear it if there is outstanding data?